cmd/root.go

```diff
@@ -36,16 +36,19 @@ func root(cmd *cobra.Command, args []string) {
 	}
 	log.Info("Start checking metrics", zap.String("prometheus", Flag.prometheusAPIURL))
 
-	for i := range Conf.Rules {
-		Conf.Rules[i].NotifyFunc = NotifyFunction
-		Conf.Rules[i].AlertFunc = AlertFunction
-	}
-
-	metricsChecker, err := metrics.NewChecker(Flag.prometheusAPIURL, Conf.Rules, Conf.Interval)
+	metricsChecker, err := metrics.NewChecker(Flag.prometheusAPIURL)
 	if err != nil {
 		log.Fatal(err.Error())
 	}
-	err = metricsChecker.Run()
+	if metricsChecker == nil {
+		return
+	}
```

Comment:
It seems we don't need to check if it is nil here.

Reply:
this only intend to make GoLand checker happy with `cmd/root.go:48` and `cmd/root.go:51`.